README.rst

```diff
@@ -495,6 +487,23 @@ Python dictionary representing a C++ variable::
       'detaileddescription: paragraphs,
     }
 
+``location`` item
+.................
+
+Python dictionary representing a location::
+
+    location = { 'file': 'filename.h', 'line-start': 10, 'line-end': 12 | None }
+
+``type0`` item
+.................
+
+Python dictionary representing a c++ type::
+
+    type = {
+      'type': 'sometype',
```

Comment:
This looks weird to me - because we get this double usage `return_info["type"]["type"]` I think it would be better to call it name. Such that we have `return_info["type"]["name"]`